user-guide/Reference/DataMiner_Tools/Data_Aggregator_Migrator.md

```diff
@@ -11,6 +11,9 @@ The Data Aggregator Migrator should be used after upgrading the Data Aggregator
 > [!TIP]
 > See also: [Data Aggregator](xref:Data_Aggregator_DxM)
 
+> [!NOTE]
+> This tool should be used after upgrading the DataAggregator DxM from 2.x.x to 3.x.x or higher.
```

Comment:
Since this note repeats what is already said in the introduction paragraph, I'm wondering if we couldn't improve that introduction instead (as I prefer not to repeat the same content exactly if we can avoid it). Perhaps we could adjust the text a bit and put some things in bold? For example:
_The Data Aggregator Migrator should be used **after** upgrading the Data Aggregator DxM from version **2.x.x to 3.x.x** or higher._

Reply:
Hi @MariekeGO, indeed it makes sense to adjust the text.

Reply:
Adjusted accordingly!